examples_tests/ComputePointCloud/rasterizer.comp

```diff
@@ -0,0 +1,63 @@
+// Copyright (C) 2018-2020 - DevSH Graphics Programming Sp. z O.O.
+// This file is part of the "Nabla Engine".
+// For conditions of distribution and use, see copyright notice in nabla.h
+
+#version 450
+
+#pragma shader_stage(compute)
+
+layout (local_size_x = 256, local_size_y = 1) in;
+
+layout (push_constant) uniform pushConstants
+{
+	uvec2 imgSize;
+	uint pointCount;
+	mat4 mvp;
+} u_pushConstants;
+
+layout (set = 0, binding = 0, r32ui) uniform coherent uimage2D outImage;
+layout (set = 0, binding = 1, std430) readonly buffer pointCloud {
+	vec3 position[];
+} u_pointCloud;
+
+void main()
+{
+	if (gl_GlobalInvocationID.x >= u_pushConstants.pointCount) 
+	{
+		return;
+	}
+	vec3 pointPosition = u_pointCloud.position[gl_GlobalInvocationID.x];
+	vec4 transformed = u_pushConstants.mvp * vec4(pointPosition, 1.0);
+
+	// Behind the camera
+	if (transformed.z < 0.0) 
```

Comment:
you might want to bitwise XOR the most significant bit of `transformed.w` with the `transformed.z` (because a negative sign in W flips post divide NDC z)